src/components/SearchBox/SearchBox.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './SearchBox.scss';
+
+class SearchBox extends Component {
+  constructor(props) {
```

Comment:
We can lose the constructor here and just assign state like this:
```jsx
class SearchBox extends Component {
    state = {
        value: '',
    }
}
```

Reply:
I'll still keep the constructor for clarity.

Reply:
What do you mean by clarity? Your constructor is literally doing nothing that can't be done without the constructor! It's just bloat at this point... 

If there's something that cant be done without the constructor then it makes sense to use the constructor

Reply:
Babel adds the constructor in the end after the transpile and injects super with all the properties (including props). For that explicit clarity I mentioned constructor to avoid confusions but I am fine with removing it.